js/lib/tree.js

```diff
@@ -106,12 +99,11 @@ var NodeView = widgets.WidgetView.extend({
             .find('i.jstree-icon.jstree-ocl').first();
     },
 
-    setOpenCloseIcon: function() {
-        var open_icon = this.getOpenIcon();
+    setOpenCloseIcon: function(recursive=false) {
```

Comment:
You added a `recursive` argument but don't seem to use it in the function (only passing it to `setOpenCloseIcon` on children)

Reply:
Thanks a lot for updating the PR!! Could you please remove the `recursive` argument? Then I'll be really happy to merge it :)

Reply:
Oh sorry, I see it is now used. LGTM

Reply:
Thank you for looking at it so quickly.
I was actually planning on using the argument, I had forgotten to add the if statement (see l. 125) checking for it. I think in some cases the recursion does not make sense, e.g. if selectively changing the icon of the parent node. That's why the default is false, so `change:close_icon` etc. will not do a recursion.